workflow/scripts/filter_report_fuseq_wes.py

```diff
@@ -0,0 +1,158 @@
+        fusiondb = int(columns[11])
+        paralog = columns[12]
+        if fusiondb == 1 or filter_on_fusiondb is False:
+            break_points = ["", ""]
+            break_point1 = ""
+            break_point2 = ""
+            if SR_support > 0:
+                if fusion_name in fusion_breakpoint_dict:
+                    break_points = fusion_breakpoint_dict[fusion_name]
+                if reverse_fusion_name in fusion_breakpoint_dict:
+                    break_points = fusion_breakpoint_dict[reverse_fusion_name]
+                break_point1 = break_points[0]
+                break_point2 = break_points[1]
+            filtered_fusions.append([fusion_name, break_point1, "", break_point2, "", paralog, SR_support, MR_support, support])
+    return filtered_fusions
+
+
+def annotate_fusion(filtered_fusions, input_gtf):
+    gene_dict = {}
+    chr_pos_dict = {}
+    transcript_exon_max = {}
+    i = 0
+    for fusion in filtered_fusions:
+        if not (fusion[1] == "" or fusion[3] == ""):
+            gene1 = fusion[0].split("--")[0]
+            gene2 = fusion[0].split("--")[1]
+            gene_dict[gene1] = ""
+            gene_dict[gene2] = ""
+            bp1_chrom = fusion[1].split(":")[0]
+            bp1_pos = (int(fusion[1].split(":")[1].split("-")[0]) + int(fusion[1].split(":")[1].split("-")[1])) / 2
+            bp2_chrom = fusion[3].split(":")[0]
+            bp2_pos = (int(fusion[3].split(":")[1].split("-")[0]) + int(fusion[3].split(":")[1].split("-")[1])) / 2
+            if bp1_chrom in chr_pos_dict:
+                chr_pos_dict[bp1_chrom].append([bp1_pos, i, 2, 100000, 0, "", ""])
+            else:
+                chr_pos_dict[bp1_chrom] = [[bp1_pos, i, 2, 100000, 0, "", ""]]
+            if bp2_chrom in chr_pos_dict:
+                chr_pos_dict[bp2_chrom].append([bp2_pos, i, 4, 100000, 0, "", ""])
+            else:
+                chr_pos_dict[bp2_chrom] = [[bp2_pos, i, 4, 100000, 0, "", ""]]
+        i += 1
+    for gtf in input_gtf:
+        columns = gtf.strip().split("\t")
+        type = columns[2]
+        if type != "CDS":
+            continue
+        gene_name = columns[8].split("gene_id \"")[1].split("\";")[0]
+        if gene_name in gene_dict:
+            chrom = columns[0]
+            pos = (int(columns[3]) + int(columns[4])) / 2
+            if chrom in chr_pos_dict:
+                for bp in chr_pos_dict[chrom]:
+                    direction = columns[6]
+                    if direction == "-":
+                        distance = bp[0] - pos
+                    else:
+                        distance = pos - bp[0]
+                    if distance < 0:
+                        distance = 100000
+                    transcript_id = columns[8].split("transcript_id \"")[1].split("\";")[0]
+                    if transcript_id == "NM_001353765":
+                        continue
+                    exon_number = int(columns[8].split("exon_number \"")[1].split("\";")[0])
```

Comment:
```suggestion
                    exon_number = int(attribute.split("exon_number \"")[1].split("\";")[0])
```

see previous comment about creating a function that generate a dict with the values